src/code/mixins/app-view.coffee

```diff
@@ -98,9 +98,10 @@ module.exports =
       else
         undo = redo = false
       if undo or redo
-        e.preventDefault()
-        @props.linkManager.redo() if redo
-        @props.linkManager.undo() if undo
+        if (@props.linkManager.undoRedoIsVisible)
+          e.preventDefault()
```

Comment:
Yes, it gets handled by CODAP.

If CODAP declares that it is using undo/redo, this cases "undoRedoIsVisible" to be false.

This has the effect of not only hiding the buttons, but also causing the button presses to be ignored. (Although actually that would happen anyway, because CODAP catches them, but better to be explicit).

It may be confusing that the property "undoRedoIsVisible" also has an effect on the keyboard presses, so we could change the name if so.